src/styles/index.scss

```diff
@@ -2,2 +1,19 @@
+@use './header.scss';
+@use './card-container';
+
+.page {
+  display: flex;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  scroll-behavior: smooth;
+}
+
+.body {
```

Comment:
The class name `.body` can be misleading because it might be confused with the body tag. Consider using a more descriptive name for this class.